virt/virt-4-13-release-notes.adoc

```diff
@@ -168,13 +173,8 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 //CNV-20965 Release note: PREVIEW Default creation and deployment of common set of instancetypes and preferences that eventually replace common templates
 
-//CNV-21922 Release note: PREVIEW (Looks like a Dev preview so maybe no doc) DevPrev: Instancetype and Preference inference based on Volume/PVC annotations
-
 //CNV-21991 Release note: PREVIEW CNV Provider for Hypershift
 
-//CNV-21994 Release note: PREVIEW Create a `VirtualMachine` definition from the CLI supplying only an instancetype with an optional set of preferences, volumes, and networks
```

Comment:
Did you mean to remove these placeholders (for CNV-21922 and CNV-21994)?

Reply:
I did mean to remove them. CNV-21992 and CNV-21994 were closed I think by Avital as Won't Do.